scripts/DickeModel.py

```diff
@@ -8,7 +8,7 @@
 from pyLIQTR.PhaseEstimation.pe import PhaseEstimation
 from networkx import path_graph, set_node_attributes, get_node_attributes, draw, draw_networkx_edge_labels
```

Comment:
Removing unused imports, not sure if needed for later, but there are a lot of imports not used, like cmathh, all of networkx, openfermion, etc.